include/UIKit/UIImage.h

```diff
@@ -92,7 +96,9 @@ UIKIT_EXPORT_CLASS
 + (UIImage*)imageNamed:(NSString*)name
                          inBundle:(NSBundle*)bundle
     compatibleWithTraitCollection:(UITraitCollection*)traitCollection STUB_METHOD;
-+ (UIImage*)imageNamed:(NSString*)name; // Note, this caches the images somewhat like iPhone OS 2ish in that it never releases them. :)
++ (UIImage*)imageNamed:(NSString*)name; // Note, this caches the images
+// somewhat like iPhone OS 2ish in
+// that it never releases them. :)
```

Comment:
This cache design didn't make much sense when I looked at the impl. Can you fix it? 

---
In reply to: [84753537](https://github.com/Microsoft/WinObjC/pull/1185#discussion_r84753537) [](ancestors = 84753537,84747216)